.github/workflows/build.yml

```
@@ -19,14 +19,17 @@ jobs:
     steps:
       - name: Check out mitiq
         uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
```

```suggestion
        uses: astral-sh/setup-uv@v6
```

[v6](https://github.com/astral-sh/setup-uv/releases/tag/v6.0.0) is out!